Assets/Scripts/Controllers/TimeManager.cs

```diff
@@ -190,4 +190,12 @@ private void InvokeEvent(Action<float> eventAction, float time)
             eventAction(time);
         }
     }
+
+    /// <summary>
+    /// Destroy this instance.
+    /// </summary>
+    public void Destroy()
```

Comment:
maybe call it Reset() ? usage looks weird in methods that should initialize the game (NewWorld, LoadWorld)